spring-cloud-netflix-eureka-server/src/test/java/org/springframework/cloud/netflix/eureka/server/EurekaInstanceMonitorTests.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2013-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.netflix.eureka.server;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.netflix.appinfo.InstanceInfo;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.search.MeterNotFoundException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Configuration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.springframework.cloud.netflix.eureka.server.FixtureEurekaInstances.getInstanceInfo;
+import static org.springframework.cloud.netflix.eureka.server.FixtureEurekaInstances.getLeaseInfo;
+
+/**
+ * @author Wonchul Heo
+ */
+@SpringBootTest(classes = EurekaInstanceMonitorTests.Application.class,
+		webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+		value = { "spring.application.name=eureka", "eureka.instance.metrics.enabled=true" })
+class EurekaInstanceMonitorTests {
+
+	private static final String APP_NAME = "MY-APP-NAME";
+
+	@Autowired
+	private InstanceRegistry instanceRegistry;
+
+	@Autowired
+	private MeterRegistry meterRegistry;
+
+	private InstanceInfo instanceInfo;
+
+	private InstanceInfo instanceInfo2;
+
+	@BeforeEach
+	void setup() {
+		this.instanceRegistry.clearRegistry();
```

Comment:
Please avoid unnecessary use of  `this.` keyword.

Reply:
I thought it was a convention because it was included in other test codes 🤣